weblate/templates/manage-access.html

```diff
@@ -39,13 +39,31 @@
   <td>{% if user.last_login %}{{ user.last_login|naturaltime }}{% else %}{% trans "Not yet signed in" %}{% endif %}</td>
   {% user_permissions user groups %}
   <td>
+
+<a href="#" data-toggle="modal" data-target="#delete_user" class="btn btn-link btn-xs red" title="{% trans "Remove" %}">
+  {% icon 'delete.svg' %}
+</a>
 <form action="{% url "delete-user" project=object.slug %}" method="post" class="inlineform">
 {% csrf_token %}
 <input type="hidden" name="user" value="{{ user.username }}" />
-<button type="submit" class="btn btn-link btn-xs red" title="{% trans "Remove" %}">
-{% icon 'delete.svg' %}
-</button>
+<div class="modal fade" tabindex="-1" role="dialog" id="delete_user">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="{% trans "Close" %}"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title">{% trans "Are you absolutely sure?" %}</h4>
+      </div>
+      <div class="modal-body">
+        This will permanently delete {{ user.username }} from this project.
```

Comment:
Please make this translatable using the [blocktrans](https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#std:templatetag-blocktrans) tag.